omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -176,9 +177,9 @@ then
     found_manage=true
 fi
 
-if [[ -e /opt/opscode-reporting/bin/opscode-reporting-ctl ]];
+#Use socket to make sure we're on the primary in an HA system
+if [[ -e /opt/opscode-reporting/bin/opscode-reporting-ctl && -S "/tmp/.s.PGSQL.5432" ]];
```

Comment:
@marcparadise That's right. Forgot that was a thing. Any thoughts on a way to proceed?

It's not critical that the information be gathered, but it's nice to have it.

What about basing the decision on something like this?

    /opt/opscode/embedded/bin/ruby -rjson -e "puts JSON.parse(STDIN.read)['private_chef']['postgresql']['external']" < /etc/opscode/chef-server-running.json

I noticed that you did this earlier to give our best effort and fail gracefully if it doesn't work.

```
    # postgres server may be external - we'll go ahead and try to gather what we can in any case, and just capture errors to the output
    # without showing them to the user
    echo "SELECT CURRENT_TIMESTAMP; SELECT * FROM pg_stat_activity;" | chef-server-ctl psql oc_erchef --as-admin --options -tA > "$tmpdir/pg_stat_activity.txt" 2>&1
```